.markdownlintrc

```diff
@@ -0,0 +1,5 @@
+{
+    "default": true,
+    "MD013": false,
```

Comment:
This line disables the rule for max line length: https://github.com/markdownlint/markdownlint/blob/master/docs/RULES.md#md013---line-length= because there were many errors and fixing all would bloat this diff. If we want this enabled, we can enable in a separate follow-up PR.